common/components/common/projects/InviteProjectToGroupModal.jsx

```diff
@@ -0,0 +1,164 @@
+// @flow
+
+import React from 'react';
+import metrics from "../../utils/metrics.js";
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+import Form from 'react-bootstrap/Form';
+import ProjectAPIUtils from '../../utils/ProjectAPIUtils.js'
+import ConfirmationModal from '../../common/confirmation/ConfirmationModal.jsx';
+import Selector from "../selection/Selector.jsx";
+import type {MyGroupData} from "../../stores/MyGroupsStore.js";
+
+
+type Props = {|
+  projectId: number,
+  groups: $ReadOnlyArray<MyGroupData>,
+  showModal: boolean,
+  handleClose: (?MyGroupData) => void
+|};
+type State = {|
+  showModal: boolean,
+  isSending: boolean,
+  message: string,
+  selectedGroup: MyGroupData,
+  showConfirmationModal: boolean
+|};
+
+/**
+ * Modal for inviting project to join Group
+ */
+
+class InviteProjectToGroupModal extends React.PureComponent<Props, State> {
+  constructor(props: Props): void {
+    super(props);
+    this.state = {
+      showModal: false,
+      isSending: false,
+      message: "",
+      positionToJoin: null,
+      roleTag: null,
+      showConfirmationModal: false
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.askForSendConfirmation = this.askForSendConfirmation.bind(this);
+    this.receiveSendConfirmation = this.receiveSendConfirmation.bind(this);
+    // this._fieldsFilled = this._fieldsFilled.bind(this);
```

Comment:
What's this for?

Reply:
Unused leftovers from copy-paste code.  Removed!